dts/bindings/regulator/regulator-i2c.yaml

```diff
@@ -63,4 +63,30 @@ properties:
         description: |
           If the enable bit should be zero to turn the regulator on, add this
           property.
-
+    current-levels:
+        type: array
+        required: false
+        description: |
+          Array of current limit values in uA, followed by the register value
+          that must be written to enable the current limit. For example,
+          [800000, 0x3] denotes a value of 0x3 must be written to the register
+          to set a current limit of 800mA
+    num-current-levels:
+        type: int
+        required: false
+        default: 0
+        description: Number of current limit levels this regulator supports
```

Comment:
You must justify all default values in this binding; see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#general-rules.

Reply:
I've added a short justification, essentially the driver uses the number of current limit levels to determine if the PMIC supports current limiting.